create_builder_test.go

```diff
@@ -478,13 +478,54 @@ func testCreateBuilder(t *testing.T, when spec.G, it spec.S) {
 			it("supports directory buildpacks", func() {
 				prepareFetcherWithBuildImage()
 				prepareFetcherWithRunImages()
-				opts.Config.Buildpacks[0].URI = "some/buildpack/dir"
+				directoryPath := "testdata/buildpack"
```

Comment:
The test as it was, was wrong. Since `"some/buildpack/dir"` didn't exist, the code saw it as a `PackageLocator` (which it shouldn't be, it should be a URILocator, because it is a path to a directory buildpack), which wouldn't work with the code currently.